set.mm

```diff
@@ -837767,6 +838014,1624 @@ have GLB (expanded version).  (Contributed by Zhi Wang,
+    fuco22 $p |- ( ph -> ( B ( U P V ) A ) = ( x e. ( Base ` C ) |->
+ ( ( B ` ( M ` x ) )
+ ( <. ( K ` ( F ` x ) ) , ( K ` ( M ` x ) ) >. ( comp ` E ) ( R ` ( M ` x ) ) )
+ ( ( ( F ` x ) L ( M ` x ) ) ` ( A ` x ) ) ) ) ) $=
+      ( vb va cop cnat co cbs cfv cco cmpt cvv eqid natrcl2 natrcl3 fuco21 wceq
+      cv wa simplrl fveq1d simplrr fveq2d oveq12d mpteq2dva fvexd mptexd ovmpod
+      wcel ) AUEUFDCNOUGHIUGFKUHUIZUILMUGPQUGEFUHUIZUIBEUJUKZBUTZPUKZUEUTZUKZVO
+      UFUTZUKZVOLUKZVPOUIZUKZWANUKVPNUKUGVPHUKKULUKUIZUIZUMBVNVPDUKZVOCUKZWBUKZ
+      WDUIZUMJSGUIUNABEFGHIJKLMNOPQRSUFUETACEFLMPQVMVMUOZUCUPADFKNOHIVLVLUOZUDU
+      PUAACEFLMPQVMWJUCUQADFKNOHIVLWKUDUQUBURAVQDUSZVSCUSZVAVAZBVNWEWIWNVOVNVKZ
+      VAZVRWFWCWHWDWPVPVQDAWLWMWOVBVCWPVTWGWBWPVOVSCAWLWMWOVDVCVEVFVGUDUCABVNWI
+      UNAEUJVHVIVJ $.
+
+    $d .* x $.  $d G x $.  $d N x $.  $d O x $.  $d P x $.  $d S x $.
+    $d X x $.
+    $( The morphism part of the functor composition bifunctor.  (Contributed by
+       Zhi Wang, 30-Sep-2025.) $)
+    fucofn22 $p |- ( ph -> ( B ( U P V ) A ) Fn ( Base ` C ) ) $=
+      ( vx co cbs cfv wfn cv cop cco cmpt ovex eqid fnmpti fuco22 fneq1d mpbiri
+      ) ACBIRFUEUEZDUFUGZUHUDUTUDUIZOUGZCUGZVABUGVAKUGZVBNUEUGZVDMUGVBMUGUJVBGU
+      GJUKUGUEZUEZULZUTUHUDUTVGVHVCVEVFUMVHUNUOAUTUSVHAUDBCDEFGHIJKLMNOPQRSTUAU
+      BUCUPUQUR $.
+
+    fuco23.x $e |- ( ph -> X e. ( Base ` C ) ) $.
+    fuco23.o $e |- ( ph -> .* = ( <. ( K ` ( F ` X ) ) , ( K ` ( M ` X ) ) >.
+                ( comp ` E ) ( R ` ( M ` X ) ) ) ) $.
+    $( The morphism part of the functor composition bifunctor.  See also
+       ~ fuco23a .  (Contributed by Zhi Wang, 29-Sep-2025.) $)
+    fuco23 $p |- ( ph -> ( ( B ( U P V ) A ) ` X ) =
+        ( ( B ` ( M ` X ) ) .* ( ( ( F ` X ) L ( M ` X ) ) ` ( A ` X ) ) ) ) $=
+      ( vx cv cfv co cop cco cbs cvv fuco22 wceq wa simpr fveq2d opeq12d adantr
+      oveq12d eqtr4d fveq12d oveq123d ovexd fvmptd ) AUHTUHUIZPUJZCUJZVIBUJZVIK
+      UJZVJOUKZUJZVMNUJZVJNUJZULZVJGUJZJUMUJZUKZUKTPUJZCUJZTBUJZTKUJZWBOUKZUJZM
+      UKDUNUJCBISFUKUKUOAUHBCDEFGHIJKLNOPQRSUAUBUCUDUEUPAVITUQZURZVKWCVOWGWAMWI
+      WAWENUJZWBNUJZULZWBGUJZVTUKZMWIVRWLVSWMVTWIVPWJVQWKWIVMWENWIVITKAWHUSZUTZ
+      UTWIVJWBNWIVITPWOUTZUTVAWIVJWBGWQUTVCAMWNUQWHUGVBVDWIVJWBCWQUTWIVLWDVNWFW
+      IVMWEVJWBOWPWQVCWIVITBWOUTVEVFUFAWCWGMVGVH $.
+  $}
+
+  ${
+    fuco22natlem1.x $e |- ( ph -> X e. ( Base ` C ) ) $.
+    fuco22natlem1.y $e |- ( ph -> Y e. ( Base ` C ) ) $.
+    fuco22natlem1.a $e |- ( ph ->
+        A e. ( <. F , G >. ( C Nat D ) <. M , N >. ) ) $.
+    fuco22natlem1.h $e |- ( ph -> H e. ( X ( Hom ` C ) Y ) ) $.
+    ${
+      fuco22natlem1.k $e |- ( ph -> K ( D Func E ) L ) $.
+      $( Lemma for ~ fuco22nat .  The commutative square of natural
+         transformation ` A ` in category ` D ` , mapped to category ` E ` by
+         the morphism part ` L ` of the functor.  (Contributed by Zhi Wang,
+         30-Sep-2025.) $)
```

Comment:
It would be nice if we had a syntax definition for triangular and square commutative diagrams, maybe this could be expressed more concisely?